app/test/frontend/golden/my_publishers.html

```diff
@@ -74,7 +74,7 @@ <h1 class="_visuallyhidden">Dart pub</h1>
     <div class="_banner-bg">
       <div class="container">
         <div class="small-banner">
-          <form class="search-bar" action="/packages">
+          <form class="search-bar" action="/my-packages">
```

Comment:
Hmm, is this right?

Reply:
Yes, we are on the "My publisher" page, and when I search for something, it was surprising to me that it went to the /dart/packages?runtime=web page, which I had in my preferences. Instead, I think it should do a search in my packages... at least that is least surprising on that location.